server/views/components/article-body/index.njk

```diff
@@ -25,6 +25,8 @@
         {% component 'tweet', { tweet: bodyPart.value } %}
       {% elif bodyPart.type === 'instagramEmbed' %}
         {% component 'instagram-embed', { instagramEmbed: bodyPart.value } %}
+      {% elif bodyPart.type === 'blockquote' %}
+        {% component 'blockquote', { body: bodyPart.value.body | safe, footer: bodyPart.value.cite | safe, modifiers: ['small'] } %}
```

Comment:
Actually, we've got [four variants](https://cardigan.wellcomecollection.org/components/detail/blockquote--small.html) now, with `--small` being the one we're using for the WP blockquotes